### DISCUSSION TITLE
List of Rustdesk instances

### DISCUSSION THREAD

**Discussion by trymeouteh** [rustdesk/rustdesk#2871] [General]
Is there a list online somewhere of the Rust Desk instances one can use?

**Comment by rustdesk** [comment]
> Is there a list online somewhere of the Rust Desk instances one can use?

Not yet. But will be.

**Comment by weironz** [comment]
really need it.

**Comment by rustdesk** [comment]
> Rust Desk instances 

do you mean https://github.com/rustdesk/rustdesk/issues/134 or something else?

**Comment by trymeouteh** [comment]
> > Rust Desk instances
> 
> do you mean #134 or something else?

No something like instances.rustdesk.com which is a list of known rustdesk instances people can use

**Comment by grummbeer** [comment]
> rustdesk instances people can use

maybe this https://github.com/rustdesk/rustdesk#free-public-servers?
